pkg/sql/opt/exec/execbuilder/tests/local/local_test: TestExecBuild_inverted_join_geospatial failed

pkg/sql/opt/exec/execbuilder/tests/local/local_test.TestExecBuild_inverted_join_geospatial [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19153939?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19153939?buildTab=artifacts#/) on master @ [a12a7cc47c28daf79f17a96fb5947df7e2e0f7ae](https://github.com/cockroachdb/cockroach/commits/a12a7cc47c28daf79f17a96fb5947df7e2e0f7ae):


```
            │
            ├── • subquery
            │   │ id: @S1
            │   │ original sql: SELECT * FROM ltable WHERE lk > 2
            │   │ exec mode: discard all rows
            │   │
            │   └── • buffer
            │       │ sql nodes: <hidden>
            │       │ regions: <hidden>
            │       │ actual row count: 0
            │       │ label: buffer 1 (q)
            │       │
            │       └── • scan
            │             sql nodes: <hidden>
            │             kv nodes: <hidden>
            │             regions: <hidden>
            │             actual row count: 0
            │             KV time: 0µs
            │             KV contention time: 0µs
            │             KV rows decoded: 0
            │             KV bytes read: 0 B
            │             KV gRPC calls: 0
            │             estimated max memory allocated: 0 B
            │             missing stats
            │             table: ltable@ltable_pkey
            │             spans: [/3 - ]
            │
            └── • subquery
                │ id: @S2
                │ original sql: (SELECT count(*) FROM q)
                │ exec mode: one row
                │
                └── • group (scalar)
                    │ sql nodes: <hidden>
                    │ regions: <hidden>
                    │ actual row count: 1
                    │
                    └── • scan buffer
                          sql nodes: <hidden>
                          regions: <hidden>
                          actual row count: 0
                          label: buffer 1 (q)
            
[03:05:19] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30237/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/inverted_join_geospatial with config local: 11 tests, 1 failures
    logic.go:4278: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30237/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/inverted_join_geospatial:408: error while processing
    logic.go:4278: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/30237/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/inverted_join_geospatial:408: too many errors encountered, skipping the rest of the input
    panic.go:635: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/a7431914f95067c51b7762545e6e8d80/logTestExecBuild_inverted_join_geospatial45669264
--- FAIL: TestExecBuild_inverted_join_geospatial (0.69s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestExecBuild_inverted_join_geospatial.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-48690

--- a/pkg/sql/opt/exec/execbuilder/testdata/inverted_join_geospatial
+++ b/pkg/sql/opt/exec/execbuilder/testdata/inverted_join_geospatial
@@ -15,7 +15,8 @@ CREATE TABLE rtable(
   geom geometry,
   rk2 string,
   PRIMARY KEY (rk1, rk2),
-  INVERTED INDEX geom_index(geom)
+  INVERTED INDEX geom_index(geom),
+  FAMILY (rk1, rk2, geom)
 )
 
 query T

